plugins/generator.md

```diff
@@ -0,0 +1,91 @@
+## Generators
+
+### Plugin generator
+
+Use the plugin generator to create a ManageIQ plugin.
+
+For example, to create a plugin called ManageIQ::MyPlugin, do the following
+
+```sh
+cd manageiq
+bundle exec rails generate manageiq:plugin ManageIQ::MyPlugin
+```
+
+This will create the directory `plugins/manageiq-my_plugin`, and you will see
+the creation progress emitted to the console:
+
+```text
+create
+create  .codeclimate.yml
+...
+```
+
+By default, the plugin will be created in the `plugins` directory, but you can
+specify an alternate directory with `--path <path>`.
```

Comment:
We might want to articulate how the path is build normally for namespacing and if that's important to follow even if it's in a different `--path`.

Reply:
We don't build the path...it's the directory into which the plugin will be created.